lms/resources/_js_config/__init__.py

```diff
@@ -349,6 +364,18 @@ def add_deep_linking_api(self):
         self._config.setdefault("filePicker", {})
         self._config["filePicker"]["deepLinkingAPI"] = config
 
+    def enable_instructor_dashboard_entry_point(self, assignment):
+        self._config["dashboard"] = {
+            "showEntryPoint": True,
+            "entryPointRPCMethod": "openDashboard",
+            "dashboardEntryPoint": {
+                "path": self._request.route_path(
+                    "dashboard.launch.assignment", id_=assignment.id
+                ),
+                "data": {},
+            },
```

Comment:
The first two options need to be propagated to the sidebar, so they need to be set in `_hypothesis_client`, not in `_config`. See https://github.com/hypothesis/client/pull/6320/files#diff-fea291e6a68bacafef88ed99be7ad4916267ab6585b42401f1f76ddcbb37706a and https://github.com/hypothesis/lms/pull/6167/files#diff-e6f7e7b4990c33f197f354288dc4d39872ffd099166b0897d9cc3a2d7a3195fbR579

The third one (`dashboardEntryPoint`) makes sense in `_config`, because it will be used by the LMS front-end only.

Reply:
Yup, my bad, updated it.